usr/share/rear/output/default/400_copy_disk_struct_files.sh

```diff
@@ -0,0 +1 @@
+../../backup/BLOCKCLONE/default/400_copy_disk_struct_files.sh
```

Comment:
Hi @gozora , I wonder why are you linking the script from `backup` here under `output`. The script uses `backup_path`, which is not even mounted in the output stage. Also, why to link it under `default` and not under `BLOCKCLONE`? Or am I missing something?